examples/hotrod/pkg/tracing/rpcmetrics/observer_test.go

```diff
@@ -109,20 +103,17 @@ func TestTags(t *testing.T) {
 		key     string
 		variant string
```

Comment:
where is it used?

Reply:
I found here https://github.com/jaegertracing/jaeger/pull/4635/files#diff-763ae75271b1c221da3048d9f551c0fc66cdb9b1b48fbed0d98753bb204c1a6cL141

Reply:
that means it's not used, that link is still initialization path